src/modules/atommodel/serializer/RadixSerializer.ts

```diff
@@ -203,7 +203,7 @@ export class RadixSerializer {
             if (!encoder.push(Buffer.from([0b1011_1111]))) { return false }
 
             for (const key of keys) {
-                if (obj[key] === undefined) {
+                if (!obj[key] || obj[key].length === 0) {
```

Comment:
Same issue here - This will fail if the value is an empty object, or the value is false.